4001-5000/LIT4723TarikaWaldaAmid.xml

```diff
@@ -66,1219 +68,47 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <change who="DR" when="2017-10-11">Created entity</change>
             <change who="DR" when="2017-11-23">Added textparts</change>
             <change who="SG" when="2019-02-25">Added bibliography and updates text part</change>
+            <change when="2024-02-02" who="CH">Updated titles, abstract and bibliography</change>
         </revisionDesc>
     </teiHeader>
     <text xml:base="https://betamasaheft.eu/">
         <body>
             <div type="bibliography">
                 <listRelation>
-               <relation name="saws:contains" active="LIT4723TarikaWaldaAmid" passive="LIT4089Introduction"/>
+                    <relation name="saws:contains" active="LIT4723TarikaWaldaAmid" passive="LIT4089Introduction"/>
                     <relation name="betmas:formerlyAlsoListedAs" active="LIT4723TarikaWaldaAmid" passive="LIT1076alMagm"/>
-               <relation name="dcterms:creator" active="LIT4723TarikaWaldaAmid" passive="PRS4711Giyorgis"/>
-               <relation name="saws:isVersionInAnotherLanguageOf" active="LIT4723TarikaWaldaAmid#t1" passive="LIT4723TarikaWaldaAmid#t2"/>
-            </listRelation>
+                    <relation name="dcterms:creator" active="LIT4723TarikaWaldaAmid" passive="PRS4711Giyorgis"/>
+                    <relation name="saws:isVersionInAnotherLanguageOf" active="LIT4723TarikaWaldaAmid#t1" passive="LIT4723TarikaWaldaAmid#t2"/>
```

Comment:
`saws:isDirectTranslationOf
` ?
(normally the `VersionInAnotherLanguageOf `relation is for parallel versions from a common source or cases when we do not know or care which is translation of which, for translations there are various Translation relations, DirectTranslation, VerbatimTranslation, CloseTranslation, LooseTranslation etc etc)